solution/ui/regulations/eregs-vite/src/views/Subjects.vue

```diff
@@ -351,8 +351,6 @@ getDocSubjects();
                 </template>
                 <template #links>
                     <HeaderLinks
-                        :about-url="aboutUrl"
-                        :resources-url="resourcesUrl"
```

Comment:
is subjectsUrl not needed here?

Reply:
Nope, not needed.  This is the Subjects page -- we don't need to link to it because we're already there.  It's the same pattern in the Statutes page: we pass in the subjects URL but not the statutes URL.